packages/react-core/src/components/Icon/examples/IconContentSizes.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const IconContentSizes: React.FunctionComponent = () => (
+  <React.Fragment>
+    <Icon isInline size="xl" iconSize="sm">
```

Comment:
did you mean to have `isInline` on these examples? Doesn't appear to be doing anything.

```suggestion
    <Icon size="xl" iconSize="sm">
```